.gitignore

```diff
@@ -2,6 +2,10 @@
 node_modules
 /dist
 
+# local env files
+.env.local
+.env.*.local
+
```

Comment:
There is already a *.local further down the file.

Reply:
I would rather keep the *.local because it allows to mark other files or even folders as local files. I use it for trivy export files for experimenting.